src/TForkedProcess.cpp

```diff
@@ -87,24 +87,25 @@ void TForkedProcess::slotReceivedData() {
     }
 }
 
-int TForkedProcess::sendMessage( lua_State * L ) {
-    QPointer<TForkedProcess> * forkedProcess = *((QPointer<TForkedProcess> **) lua_topointer(L, lua_upvalueindex(1)));
+int TForkedProcess::sendMessage(lua_State* L)
+{
+    QPointer<TForkedProcess>* forkedProcess = *((QPointer<TForkedProcess>**)lua_topointer(L, lua_upvalueindex(1)));
```

Comment:
This seems a little odd to me,  a `QPointer<TForkedProcess> *` - which is effectively a pointer to a pointer? :open_mouth: